packages/orbit-components/src/Breadcrumbs/BreadcrumbsItem/types.d.ts

```diff
@@ -7,7 +7,7 @@ import type * as Common from "../../common/types";
 
 export interface Props extends Common.Globals {
   readonly active?: boolean;
-  readonly component?: Common.Component;
+  readonly component?: React.ElementType;
```

Comment:
The previous Common.Component included three things:
- string
- React.Component<any>
- React.ForwardRefRenderFunction<HTMLElement, any>

I am concerned if this change is retrocompatible with all the above ones.

If it does, why not just change the value of Common.Component?